network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/HvdcLineImpl.java

```diff
@@ -91,7 +99,9 @@ public double getActivePowerSetpoint() {
 
     @Override
     public HvdcLine setActivePowerSetpoint(double activePowerSetpoint) {
+        double oldValue = resource.getAttributes().getActivePowerSetpoint();
         resource.getAttributes().setActivePowerSetpoint(activePowerSetpoint);
+        index.notifyUpdate(this, "activePowerSetpoint", oldValue, activePowerSetpoint);
```

Comment:
notifyUpdate with variantId to call here

Reply:
Corrected